server/src/main/scala/controllers/Application.scala

```diff
@@ -552,47 +553,51 @@ class Application @Inject()(implicit val config: Configuration,
                                           ): Action[AnyContent] = Action.async {
 
     log.info(s"Exports: For point in time ${SDate(pointInTime.toLong).toISOString()}")
+    val portCode = airportConfig.portCode
+    val pit = MilliDate(pointInTime.toLong)
+
+    val fileName = f"$portCode-$terminalName-desks-and-queues-${pit.getFullYear()}-${pit.getMonth()}%02d-${pit.getDate()}%02dT" +
+      f"${pit.getHours()}%02d-${pit.getMinutes()}%02d-hours-$startHour%02d-to-$endHour%02d"
 
-    val crunchStateFuture: Future[Option[CrunchState]] = loadBestCrunchStateForPointInTime(pointInTime.toLong)
 
-    exportDesksToCSV(pointInTime, terminalName, crunchStateFuture, startHour, endHour)
+    exportDesksToCSV(terminalName, pit, startHour, endHour).map {
+      case Some(csvData) =>
+        Result(
+          ResponseHeader(200, Map("Content-Disposition" -> s"attachment; filename='$fileName.csv'")),
+          HttpEntity.Strict(ByteString(csvData), Option("application/csv"))
+        )
+      case None =>
+        NotFound("Could not find desks and queues for this date.")
+    }
   }
 
   def exportDesksToCSV(
-                        pointInTime: String,
                         terminalName: TerminalName,
-                        crunchStateFuture: Future[Option[CrunchState]],
+                        pointInTime: MilliDate,
                         startHour: Int,
                         endHour: Int
-                      ): Future[Result] = {
-    val pit = MilliDate(pointInTime.toLong)
+                      ): Future[Option[String]] = {
 
-    log.info(s"Start hour: $startHour End hour: $endHour")
 
-    val portCode = airportConfig.portCode
+    log.info(s"Start hour: $startHour End hour: $endHour")
```

Comment:
If we want to keep this log line then maybe add some context as to what it's referring to?